packages/vant/src/area/test/index.spec.ts

```diff
@@ -7,6 +7,24 @@ import {
   PickerConfirmEventParams,
 } from '../../picker';
 
+test('should columns placeholder displayed normally', async () => {
+  const columnsPlaceholder = ['省份', '省份', '区县'];
```

Comment:
Prefer using English in test files:

```suggestion
  const columnsPlaceholder = ['Province', 'City', 'County'];
```